graph_annotation.go

```diff
@@ -20,10 +20,21 @@ type GraphAnnotation struct {
 }
 
 // CreateGraphAnnotation creates graph annotation.
-func (c *Client) CreateGraphAnnotation(annotation *GraphAnnotation) error {
+func (c *Client) CreateGraphAnnotation(annotation *GraphAnnotation) (*GraphAnnotation, error) {
 	resp, err := c.PostJSON("/api/v0/graph-annotations", annotation)
 	defer closeResponse(resp)
-	return err
+
+	if err != nil {
+		return nil, err
+	}
+
+	var createdAnnotation GraphAnnotation
```

Comment:
[nit]
In Go, shorter name is preferred especially if the lifetime is short.
I think 'annotation' or something is enough in this case.  but you can ignore this, it's ok as it is.